python/lsst/summit/extras/focusAnalysis.py

```diff
@@ -150,11 +153,18 @@ def _bboxToMplRectangle(self, bbox, colorNum):
         return rectangle
 
     @staticmethod
-    def gauss(x, *pars):
+    def gauss(x: float, *pars: float) -> float:
```

Comment:
`x` is indeed a float. Is that how you annotate catching a list of other parameters though?

Reply:
Yes, is done that way for annotating a list of parameters.